src/libraries/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -1877,6 +1877,7 @@ public void Process_StartInvalidNamesTest()
             Assert.Throws<Win32Exception>(() => Process.Start("exe", string.Empty, new SecureString(), "thisDomain"));
         }
 
+        [OuterLoop("May take many seconds the first time it's run")]
```

Comment:
Why?

Reply:
Not entirely sure.  I'm assuming it has something to do with my domain, or VPN, or something.  When I first run the test suite, it takes almost 30 seconds for this one test.  If I then run it again, it's less than a second.  I've not profiled to see where the time is being spent.